examples/hello-world/hello-tabular-stats/README.md

```diff
@@ -0,0 +1,185 @@
+# Tabular Data Federated Statistics 
+
+In this example, we will show how to generate federated statistics for tabular data that can be represented as Pandas Data Frame.
+
+## Setup NVFLARE
+Follow the [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html) to setup virtual environment and install NVFLARE
+
+Let's first install required packages.
+
+```
+pip install --upgrade pip
+
+cd NVFlare/examples/hello-world/hello-tabular-stats
+
+pip install -r requirements.txt
+```
+
+
+## Install Optional Quantile Dependency -- fastdigest
+
+If you intend to calculate quantiles, you need to install fastdigest. 
+
+Skip this step if you don't need quantile statistics. 
+
+```
+pip install fastdigest==0.4.0
+```
+
+on Ubuntu, you might get the following error:
+
+  Cargo, the Rust package manager, is not installed or is not on PATH.
+  This package requires Rust and Cargo to compile extensions. Install it through
+  the system's package manager or via https://rustup.rs/
+      
+  Checking for Rust toolchain....
+
+This is because fastdigest (or its dependencies) requires Rust and Cargo to build. 
+
+You need to install Rust and Cargo on your Ubuntu system. Follow these steps:
+Install Rust and Cargo
+Run the following command to install Rust using rustup:
+
+```
+./install_cargo.sh
+```
+
+Then you can install fastdigest again
+```
+pip install fastdigest==0.4.0
+```
+
+## Code Structure
+first get the example code from github:
+
+navigate to the hello-tabular-stats directory:
+
+```
+git switch <release branch>
```

Comment:
do we ask this step now? Why not include nvflare in the requirements.txt?

Reply:
Seems most requirement.txt in hello world examples do list nvflare, .e.g https://github.com/NVIDIA/NVFlare/blob/main/examples/hello-world/hello-lightning/requirements.txt

Reply:
I would prefer is users just run `pip install -r requirements.txt` and all requirements are installed.